client/src/lobby/createGame.ts

```diff
@@ -263,9 +263,37 @@ function submit() {
   // if this is a new game or a change of options
   const isNew = $("#create-game-table-number").val() === "";
 
-  // We need to mutate some values before sending them to the server
-  const timeBaseMinutes = getTextboxForTimeBase("createTableTimeBaseMinutes");
-  const timeBaseSeconds = Math.round(timeBaseMinutes * 60); // The server expects this in seconds
+  // Get timer values
+  const timed = getCheckbox("createTableTimed");
+  let timeBaseSeconds = 0;
+  let timePerTurn = 0;
+
+  // Try getting the user's values - if they are invalid, leave the dialog open
+  if (timed) {
+    let timeValue: number;
+    let foundErrors = false;
+
+    try {
+      timeValue = getTextboxForTimeBase("createTableTimeBaseMinutes");
+      timeBaseSeconds = Math.round(timeValue * 60);
+    } catch (err) {
+      // invalid value, inform the UI and do not close the tooltip
+      $("#createTableTimeBaseMinutes").addClass("wrongInput");
+      foundErrors = true;
+    }
+
+    try {
+      timePerTurn = getTextboxForTimePerTurn("createTableTimePerTurnSeconds");
+    } catch (err) {
+      // invalid value, inform the UI and do not close the tooltip
```

Comment:
use capital letters for comments, i.e. match the existing style for all of the other code